index.html

```diff
@@ -0,0 +1,25 @@
+<html>
+
+<!-- View this file with `npm run web` -->
+
+<head>
+  <meta content="text/html;charset=utf-8" http-equiv="Content-Type" />
+  <script type="module">
+    import {createPreprocessor} from './pkg/standalone/standalone.js';
+
+    async function run() {
```

Comment:
this is how I'll be using it in my REPL, tho it'll be `import { createPreprocessor } from 'content-tag/standalone'`